addons/crm_iap_lead/models/crm_iap_lead_mining_request.py

```diff
@@ -163,7 +193,13 @@ def _prepare_iap_payload(self):
             payload.update({'company_size_min': self.company_size_min,
                             'company_size_max': self.company_size_max})
         if self.industry_ids:
-            payload['industry_ids'] = self.industry_ids.mapped('reveal_id')
+            # accumulate all reveal_ids (separated by ',') into one list
+            # eg: 3 records with values: "175,176", "177" and "190,191"
+            # will become ['175','176','177','190','191']
+            all_industry_ids = list(itertools.chain(*[
```

Comment:
Can't we simply make a list comprehension ? Like
```
all_industry_ids = [
    reveal_id.strip()
    for reveal_ids in self.mapped('industry_ids.reveal_ids')
    for reveal_id in reveal_ids.split(',')
]
```

Reply:
Side remark: added the strip() just because I bet people will add a void space somewhere

Reply:
And here I was, proud to have used itertools! :D